scripts/provisioning/setupcmd.py

```diff
@@ -33,8 +33,7 @@ class SetupCmd(object):
   ldap_passwd = None
   rootdn_passwd = None
   cluster_id = None
-  server_nodes_count = 0
-  hosts_list = None
+  machine_id = None
```

Comment:
Need to write s3_provisioner_utils common class

Reply:
taken care by @sachitanands in his PR.